loading-indicator.common.ts

```diff
@@ -0,0 +1,26 @@
+interface AndroidOptionsI {
```

Comment:
The `I` is somewhat typical of an interface. I kind of prefer it. If big deal to you, I can change it. Not 100% sure what core team does but I've seen it in various developments. I think it does help the understanding.